recipes/matching/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "matching" %}
+{% set version = "1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 62fa1c2dc1b36f2ea70584d5ca776b3af02e9516140ee4c0c88123c7abe66896
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
```

Noarch will need a Python version range here:
```suggestion
    - python >=3.5
    - pip
  run:
    - python >=3.5
```